sdk/python/kfp/components/_python_op.py

```diff
@@ -58,8 +61,36 @@ def __init__(self, type=None):
 #OutputFile[GcsPath[Gzipped[Text]]]
 
 
-class OutputFile(Generic[T], str):
-    pass
+class OutputPath:
+    '''When creating component from function, OutputPath should be used as function parameter annotation to tell the system that the function wants to output data by writing it into a file with the given path instead of returning the data from the function.'''
+    def __init__(self, type=None):
+        self.type = type
```

Comment:
type?

Reply:
It's the input/output type.

Before
```def func(data: list):```
after
```def func(data_file: InputFile(list)):```